### ISSUE TITLE
`lines-around-comment` is fooled by two block comments on same line

### ISSUE THREAD

**Issue by IanVS** [eslint/eslint#3509]
With the code:

```js
/*eslint lines-around-comment: [2, { "beforeBlockComment": true, "afterBlockComment": true }]*/

var night = "long";
/* what a great and wonderful day */
var day = "great"
```

I get two errors:

```js
  4:1  error  Expected line before comment  lines-around-comment
  4:1  error  Expected line after comment   lines-around-comment
```

However, if I add a second block comment to the same line, the errors disappear.

```js
/*eslint lines-around-comment: [2, { "beforeBlockComment": true, "afterBlockComment": true }]*/

var night = "long";
/* what a great and wonderful day */ /* another block comment */
var day = "great"
```

This is on current master.

**Comment by IanVS** [comment]
This is likely due to the [inline comment exception](http://eslint.org/docs/rules/lines-around-comment#inline-comments).  But, is this expected behavior if there are only comments on the line?

**Comment by nzakas** [comment]
That looks like a bug to me. I'd expect to see those same errors on the second example.

**Comment by alberto** [comment]
@nzakas If we where to fix this, should we report two or four errors (two errors for each of the comment blocks)?

**Comment by nzakas** [comment]
Four seems right to me
